formats/finder_email_signup/publisher/details.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+   "required": [
+     "details"
+   ],
+  "properties": {
+     "details": {
```

Comment:
The details schema shouldn't itself contain details; it is included as the details key in the parent.